sale_order_line_price_history/wizards/sale_order_line_price_history.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 Tecnativa - Ernesto Tejeda
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="sale_order_line_price_history_view_form" model="ir.ui.view">
+        <field name="name">sale.order.line.price.history.view.form</field>
+        <field name="model">sale.order.line.price.history</field>
+        <field name="arch" type="xml">
+            <form>
+                <group col="4">
+                    <field name="partner_id"/>
+                    <field name="product_id"/>
+                    <field name="include_commercial_partner"/>
+                    <field name="include_quotations"/>
+                </group>
+                <group string="Sale history">
+                    <field name="line_ids" nolabel="1" readonly="1">
+                        <form>
+                            <div class="oe_button_box" name="button_box">
+                                <button class="oe_stat_button"
+                                        string="Set price"
+                                        type="object"
+                                        name="action_set_price"
+                                        icon="fa-check"/>
+                            </div>
+                            <group>
+                                <field name="sale_order_line_id" invisible="1"/>
+                                <field name="history_sale_order_line_id" invisible="1"/>
+                                <field name="order_id"/>
+                                <field name="partner_id"/>
```

Comment:
```suggestion
                                <field name="partner_id" attrs="{'column_invisible': [('parent.partner_id', '=', False)]}"/>
```
With this we can remove the first statement in ROADMAP section:
https://github.com/OCA/sale-workflow/pull/1082/files#diff-93b55544ac6a86454091405656f9ca6dR55